meta-balena-common/recipes-containers/mkfs-hostapp-native/files/create.ext4

```diff
@@ -8,7 +8,7 @@ balenad -s=@BALENA_STORAGE@ --data-root="$SYSROOT/balena" -H unix:///var/run/bal
 pid=$!
 sleep 5
 
-hostapp-update -f /input -n
+hostapp-update -f /input -n -s
```

Comment:
By moving the logic to update-balena-supervisor this change is not needed as hostapp-update won't change. The supervisor would call both update-balena-supervisor and hostapp-update, effectively replacing the balenaHUP proxy.